common/djangoapps/tedix_ro/models.py

```diff
@@ -169,4 +165,11 @@ class Question(models.Model):
 class StudentReportSending(models.Model):
     course_id = CourseKeyField(max_length=255)
     user = models.ForeignKey(User, related_name='stident_report_sending', on_delete=models.CASCADE)
-    grade = models.FloatField()
+
+
+class StudentCourseReport(models.Model):
+    student = models.ForeignKey(StudentProfile, related_name='course_reports', on_delete=models.CASCADE)
+    course_due_date = models.ForeignKey(StudentCourseDueDate, null=True, related_name='course_reports', on_delete=models.CASCADE)
```

Comment:
I'm not sure but maybe you shouldn't delete reports if course due date object is deleted? I mean I think it would be safer to use `on_delete=models.SET_NULL` for `course_end_date` field instead of `on_delete=models.CASCADE`

Reply:
only a superuser can delete StudentCourseReport. I think we can leave it.